README.md

```diff
@@ -471,6 +472,28 @@ jobs:
           password: ${{ secrets.QUAY_ROBOT_TOKEN }}
 ```
 
+### DigitalOcean Container Registry
+Use your DigitalOcean registered email address and an API access token to authenticate.
```

Comment:
missing new line before and after this one

Reply:
Fixed the formatting issue!